frontend/public/components/daemonset.jsx

```diff
@@ -32,7 +32,7 @@ const DaemonSetRow = ({obj: daemonset}) => <ResourceRow obj={daemonset}>
     </Link>
   </div>
   <div className="col-lg-3 hidden-md hidden-sm hidden-xs">
-    <Selector selector={daemonset.spec.selector} />
+    <Selector selector={daemonset.spec.selector} namespace={daemonset.metadata.namespace} />
```

Comment:
Unfortunately, `WorkloadListRow` doesn't work for daemon sets because the pods column is wrong, so I have to make the fix here, too.